pygui/renderers/pygfx_renderer.py

```diff
@@ -0,0 +1,75 @@
+from collections import defaultdict
+
+import pygfx as gfx
+
+from . import Renderer
+
+
+class MeshEvents(gfx.Mesh):
+    """Custom subclass of gfx.Mesh to enable handling of events."""
```

Comment:
Do we have a ticket on pygfx yet, for an event system? Or at least to think about it?

Reply:
Great idea: here it is: https://github.com/pygfx/pygfx/issues/255